src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -52,6 +52,16 @@ export const AnalyticsFunnel = (props: AnalyticsFunnelProps) => {
   return <InnerAnalyticsFunnel {...props} />;
 };
 
+const funnelCancelled = ({ funnelInteractionId }: { funnelInteractionId: string }) => {
```

Comment:
To me `funnelCancelled` and `funnelComplete` sound generic and requires reading thought the code to see what it does. What about a more specific name, e.g. `dispatchFunnelComplete` or `dispatchFunnelCompleteEvent`?

Reply:
I guess the `funnelCancelled` `funnelComplete` sounds generic and misleading to 'to cancel/complete the funnel'. Rename to `onFunnelComplete` and `onFunnelCancelled`, because the function is not only dispatching the event but also run FunnelMetrics function.